module/apmgorilla/middleware.go

```diff
@@ -147,3 +147,8 @@ func WithPanicPropagation() Option {
 		o.panicPropagation = true
 	}
 }
+
+// ServerRequestName returns a ServerOption to resolve gorilla routes
+func ServerRequestName() apmhttp.ServerOption {
+	return apmhttp.WithServerRequestName(routeRequestName)
+}
```

Comment:
Rather than exporting a new option, how about we just export `routeRequestName` (i.e. rename to `RouteRequestName`)?

Reply:
Done, thx for review